show/syslog.py

```diff
@@ -51,3 +62,69 @@ def syslog(db):
         body.append(row)
 
     click.echo(format(header, body))
+
+@syslog.command(
+    name='rate-limit-host'
+)
+@clicommon.pass_db
+def rate_limit_host(db):
+    """ Show syslog rate limit configuration for host """
+
+    header = [
+        "INTERVAL",
+        "BURST",
+    ]
+    body = []
+    syslog_configs = db.cfgdb.get_table(SYSLOG_CONFIG_TABLE)
+    if SYSLOG_CONFIG_GLOBAL_KEY in syslog_configs:
+        entry = syslog_configs[SYSLOG_CONFIG_GLOBAL_KEY]
+        body.append([entry.get(SYSLOG_RATE_LIMIT_INTERVAL, 'N/A'),
+                    entry.get(SYSLOG_RATE_LIMIT_BURST, 'N/A')])
+
+    click.echo(format(header, body))
+
+
+@syslog.command(
+    name='rate-limit-container'
+)
+@click.argument('service_name', metavar='<service_name>', required=False)
+@clicommon.pass_db
+def rate_limit_container(db, service_name):
+    """ Show syslog rate limit configuration for containers """
+
+    header = [
+        "SERVICE",
+        "INTERVAL",
+        "BURST",
+    ]
+    body = []
+    features = db.cfgdb.get_table(FEATURE_TABLE)
+    syslog_configs = db.cfgdb.get_table(SYSLOG_CONFIG_FEATURE_TABLE)
+
+    if service_name:
+        if service_name not in features:
+            raise click.UsageError('Invalid service name {}, please choose from: {}'.format(service_name, ','.join(features.keys())))
+
+        state = features[service_name].get('state')
+        if state in ['disabled', 'always_disabled']:
+            raise click.ClickException('Service {} is disabled, please enable it first'.format(service_name, ))
+
+        support_rate_limit = features[service_name].get('support_syslog_rate_limit', '').lower() == 'true'
+        if not support_rate_limit:
+            raise click.ClickException('Service {} does not support syslog rate limit'.format(service_name))
+
+        service_list = [service_name]
+    else:
+        service_list = [name for name, service_config in features.items() if service_config.get('support_syslog_rate_limit', '').lower() == 'true' and \
+                                                                             service_config.get('state') not in ['disabled', 'always_disabled']]
+
+    for service in natsorted(service_list):
+        if service in syslog_configs:
+            entry = syslog_configs[service]
+            body.append([service,
+                        entry.get(SYSLOG_RATE_LIMIT_INTERVAL, 'N/A'),
```

Comment:
seems the logic is not consistence, when service_name provided, if service does not support changing syslog rate limit, the CLI will only give a prompt msg, but here will show 'N/A'

Reply:
If service name is provided and service does not support changing syslog rate limit, the function returns here:
```
support_rate_limit = features[service_name].get('support_syslog_rate_limit', '').lower() == 'true'
        if not support_rate_limit:
            raise click.ClickException('Service {} does not support syslog rate limit'.format(service_name))
```
If service name is not provided, services who do not support changing syslog rate limit is filtered here:

```
service_list = [name for name, service_config in features.items() if service_config.get('support_syslog_rate_limit', '').lower() == 'true' and \
                                                                             service_config.get('state') not in ['disabled', 'always_disabled']]
```